lib/easypost/payment_log_report.rb

```diff
@@ -0,0 +1,4 @@
+module EasyPost
+  class PaymentLogReport < Report
```

Comment:
If we are going to have these classes wouldn't it make sense to override the methods and pass the correct type to all?